modules/ROOT/pages/prereq-aws-terraform.adoc

```diff
@@ -18,12 +18,13 @@ To install Anypoint Platform PCE on AWS:
 * When you create your AWS environment, the following resources are created:
 +
 [%header%autowidth.spread]
+.AWS-Created Resources
```

Comment:
Add a third column for the optional monitoring nodes with the following values for the following rows:
- m5.8xlarge | +3
- root disk @ 500 iops | +3
- EBS volume @ 1500 iops | +12

Use +0 for the rest of the rows

Reply:
Added the following:

| AWS Resource | Number Required (4-node) | Number Required (7-node) | Anypoint Monitoring (Optional)
| m5.8xlarge | 4 | 7 | 3
| root disk @ 500 iops | 4 | 7 | 3
| EBS volumes @ 1500 iops | 8 | 14 | 12
| EBS volume @ 3000 iops | 8 | 14 | 0
| Amazon ELB | 1 | 1 | 0
| t2.medium | 1 | 1 | 0